### PR TITLE
Add bedops sort

### PR DESCRIPTION
**Pull request by mvdbeek**
which should be a little faster than gnu sort and can be memory limited (which bedtools sort is not)

FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/bedops/sort-bed.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<tool id="bedops-sort-bed" name="sort-bed" version="2.4.41" profile="22.01">
```
Comment (bernt-matthias) [inline tools/bedops/sort-bed.xml:1]: Version token would be good.

Comment (bernt-matthias) [inline tools/bedops/sort-bed.xml:1]: Can you add "bedops" to the name?

File: tools/bedops/sort-bed.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    <command><![CDATA[
+sort-bed 
+--max-mem "\${GALAXY_MEMORY_MB:-1024}M"
+--tmpdir "\${_GALAXY_JOB_TMP_DIR:-.}"
+$unique
+$duplicates
+#for file in $inputs:
+    '$file'
+#end for
+> $output
```
Comment (bernt-matthias) [inline tools/bedops/sort-bed.xml:14]: Output should also be quoted.

File: tools/bedops/sort-bed.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    ]]></command>
+    <inputs>
+        <param name="inputs" type="data" format="bed" multiple="true"/>
```
Comment (bernt-matthias) [inline tools/bedops/sort-bed.xml:17]: Is multiple intended?

File: tools/bedops/.shed.yml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+categories:
```
Comment (bernt-matthias) [inline tools/bedops/.shed.yml:1]: Hoping that there will more tools from bedops.. How about defining the shed yaml file as suite?

Comment (mvdbeek) [reply tools/bedops/.shed.yml:1]: When we need it ?

File: tools/bedops/sort-bed.xml (hunk 3 above)
Comment (mvdbeek) [reply tools/bedops/sort-bed.xml:17]: Yes

File: tools/bedops/sort-bed.xml (hunk 1 above)
Comment (mvdbeek) [reply tools/bedops/sort-bed.xml:1]: I don't like that it doesn't match the command line, but I see we also do that with bedtools :(

**Review by bgruening — APPROVED** [review]

**Pull request merged**